www-apps/webdavcgi/webdavcgi-1.1.2.ebuild

```diff
@@ -1,20 +1,20 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-inherit eutils toolchain-funcs versionator
+inherit eutils toolchain-funcs
```

Comment:
Are we using eutils?

Reply:
No, thanks for spotting, `eutils` was removed form `inherit`.